### ISSUE TITLE
[BUG] - Unknown Error

### ISSUE THREAD

**Issue by rononline** [oven-lab/tuya_cloud_map_extractor#9]
```
None
Traceback (most recent call last):
  File "/config/custom_components/tuya_cloud_map_extractor/config_flow.py", line 80, in async_step_user
    headers, image = await validate(self.hass, user_input)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/tuya_cloud_map_extractor/config_flow.py", line 230, in validate
    return await hass.async_add_executor_job(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/tuya_cloud_map_extractor/tuya_vacuum_map_extractor/main.py", line 128, in get_map
    image = render_layout(raw_map=mapDataArr, header=header, colors=colors)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/tuya_cloud_map_extractor/tuya_vacuum_map_extractor/main.py", line 68, in render_layout
    array = to_array_custom0(pixellist, width, height, colors)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/tuya_cloud_map_extractor/tuya_vacuum_map_extractor/custom0.py", line 42, in to_array_custom0
    pixel = colors[pixeltype]
            ~~~~~~^^^^^^^^^^^
KeyError: None
```

**Comment by oven-lab** [comment]
Currently working on this issue. Check issue #7 for more info.

**Comment by oven-lab** [comment]
Try out release [v1.2.0b3](https://github.com/oven-lab/tuya_cloud_map_extractor/releases/tag/v1.2.0b3). Make sure to check show beta releases if redownloading via HACS.

**Comment by rononline** [comment]
I tried it:

```
Logger: custom_components.tuya_cloud_map_extractor.config_flow
Source: custom_components/tuya_cloud_map_extractor/config_flow.py:66
Integration: Tuya Cloud Map Extractor ([documentation](https://github.com/oven-lab/tuya_cloud_map_extractor/blob/main/README.md), [issues](https://github.com/oven-lab/tuya_cloud_map_extractor/issues))
First occurred: 21:36:16 (2 occurrences)
Last logged: 21:36:23

list index out of range
Traceback (most recent call last):
  File "/config/custom_components/tuya_cloud_map_extractor/config_flow.py", line 66, in async_step_user
    data = create_entry_data(
           ^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/tuya_cloud_map_extractor/config_flow.py", line 238, in create_entry_data
    if not (CONF_ROOM_COLOR + str(rooms[0]["ID"])) in data:
                                  ~~~~~^^^
IndexError: list index out of range
```

**Comment by oven-lab** [comment]
Should be fixed in v1.2.0b5

**Comment by rononline** [comment]
Confirm it's fixed! And it's working now!
